tests/test.py

```diff
@@ -73,8 +96,8 @@ def module_installed(module: str) -> bool:
 T = TypeVar("T")
 
 
-def hashable_skip(obj: T, skips: Set[str]) -> T:
-    """Return a hashable copy of `obj` with all keys/attributes in `skips`
+def canonical(obj: T, skips: Set[str]) -> Union[T, Dict, Tuple]:
```

Comment:
what is the significance of this change?

Reply:
Just the name, not using hashing any more so the name didn't make sense. Name now is more appropriate for what it does, create a "canonical" version of a JSON for sorting.